### PR TITLE
[MI-179] Add PHP version support to the README

### PR DESCRIPTION
**Pull request by joseconsador**
Add PHP version support to the README.
PHP5.5+ is a requirement for Guzzle 6.

/cc @miogalang @jwswj @mmolina @atroche

### References
 - Jira link: https://zendesk.atlassian.net/browse/MI-179

### Risks
 - None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 This client **only** supports Zendesk's API v2.  Please see our [API documentation](http://developer.zendesk.com) for more information.
 
+## Requrements
+* PHP (at least 5.5)
```
Comment (jwswj) [inline README.md]: PHP 5.5+

Comment (joseconsador) [inline README.md]: :haircut:

**Comment by jwswj** [comment]
:+1:

**Comment by miogalang** [comment]
:+1:

**Pull request merged**